cla-backend-go/cmd/generate_compound_attribute/main.go

```diff
@@ -0,0 +1,160 @@
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+	log "github.com/communitybridge/easycla/cla-backend-go/logging"
+)
+
+var awsSession = session.Must(session.NewSession(&aws.Config{}))
+var events EventInterface
+var stage string
+
+const (
+	tableName = "cla-%s-events"
+	attr1     = "event_company_sfid"
+	attr2     = "event_cla_group_id"
+	newAttr   = "company_sfid_cla_group_id"
+	sep       = "#"
+)
+
+type EventInterface interface {
+	FetchAndUpdateDocuments(ctx context.Context) error
+	InsertCompoundAttribute(ctx context.Context, event EventModel) error
+}
+
+type Config struct {
+	tableName      string
+	dynamoDBClient *dynamodb.DynamoDB
+	stage          string
+}
+
+type EventModel struct {
+	EventID          string `json:"event_id"`
+	EventCompanySFID string `json:"event_company_sfid"`
+	EventCLAGroupID  string `json:"event_cla_group_id"`
+}
+
+func (c Config) FetchAndUpdateDocuments(ctx context.Context) error {
```

Comment:
It is fetching all events which are having existing event_company_sfid and event_cla_group_id fields not null anddoes not exist company_sfid_cla_group_id field.
It will combine both attributes and update each and every events.